consensus/istanbul/config_test.go

```diff
@@ -126,3 +127,41 @@ func TestIsQBFTConsensusAt(t *testing.T) {
 		}
 	}
 }
+
+func TestGetValidatorContractAddress(t *testing.T) {
+	address, address1, address2, address3 := common.Address{}, common.Address{0x2}, common.Address{0x4}, common.Address{0x6}
+
+	config := *DefaultConfig
```

Comment:
here it's *, so it will copy the value, not the pointer, so DefaultConfig is not modified.
Proper fix would be to make DefaultConfig not a pointer but the value directly.